src/main/webapp/style.css

```diff
@@ -0,0 +1,24 @@
+.content {
+    font-family: Arial, Helvetica, sans-serif;
+    color: black;
+    justify-content: flex-start;
+    margin: 0;
+    padding: 0;
+}
+
+.content h1 {
+    text-align: center;
+}
+
+#searchForm {
+    margin: auto;
+    text-align: center;
+}
+
+.promoCodeList {
+    text-align: center;
+    margin-top: 100px
+}
+.promoCodeList table{
+    display: inline;
+}
```

Comment:
Is there a new line at the end of file? I can't tell with github diffs.

Reply:
Yes github only shows changes